lib/ProgramCall.js

```diff
@@ -83,21 +95,28 @@ class ProgramCall {
    * @param {object} [options]
    */
   addReturn(data, type, options = null) {
-    this.xml += iXml.iXmlNodeReturnOpen();
+    if (!type) {
+      deprecate('defaulting return type to 1024a has been deprecated. You should specify a type instead.');
```

Comment:
Again, iPgm.addReturn should warn and set type to `1024a` to maintain compatibility.

Reply:
Again will add `type = '1024a'` here as well